.env.example

```diff
@@ -29,6 +29,7 @@ NEW_MANDATORY_QUESTION_JOURNEY_ENABLED=newMandatoryQuestionJourneyEnabled
 NOTIFICATION_UNSUBSCRIBE_JWT_EXPIRY_TIME=7d
 VERIFICATION_JWT_EXPIRY_TIME=7d
 APPLICANT_HOST=http://localhost:3001/apply/applicant
+ADMIN_HOST=http://localhost:3000/apply/admin
```

Comment:
Infra pr for this var is in the ticket - needs to be merged before this

Reply:
PR: https://eu-west-2.console.aws.amazon.com/codesuite/codecommit/repositories/find-a-grant-infra/pull-requests/272/changes?region=eu-west-2

Reply:
Might be worth linking this in one of the channels for eyes

Reply:
Nice! Got it linked in the Jupiter channel